apps/explorer/src/pages/address-result/AddressBalanceBreakdown.tsx

```diff
@@ -0,0 +1,226 @@
+    const [open, setOpen] = useState(true);
+    const {
+        data: balance,
+        isLoading: isLoadingBalance,
+        isError: isBalanceErrored,
+    } = useBalance(address);
+
+    const [totalAvailableBalance, symbol] = useFormatCoin({
+        balance: balance?.totalBalance,
+    });
+
+    const {
+        data: delegatedStake,
+        isLoading: isLoadingDelegatedStakes,
+        isError: isDelegatedStakeErrored,
+    } = useGetDelegatedStake({
+        address,
+    });
+    const delegatedStakes = delegatedStake ? formatDelegatedStake(delegatedStake) : [];
+    const totalDelegatedStake = useTotalDelegatedStake(delegatedStakes);
+    const [formattedDelegatedStake] = useFormatCoin({
+        balance: totalDelegatedStake,
+    });
+
+    const {
+        data: timelockedStakedObjects,
+        isLoading: isLoadingTimelockedStakeObjects,
+        isError: isTimelockedStakedObjectsErrored,
+    } = useGetTimelockedStakedObjects(address);
+
+    const extendedDelegatedTimelockedStakes = formatDelegatedTimelockedStake(
+        timelockedStakedObjects || [],
+    );
+
+    const totalTimelockedStaked = extendedDelegatedTimelockedStakes?.reduce(
+        (acc, extendedStake) => acc + BigInt(extendedStake.principal),
+        0n,
+    );
+
+    const [formattedTimelockedStake] = useFormatCoin({
+        balance: totalTimelockedStaked,
+    });
+
+    const {
+        data: timelockedObjects,
+        isLoading: isTimelockedObjectsLoading,
+        isError: isTimelockedObjectsError,
+    } = useGetAllOwnedObjects(address, {
+        StructType: TIMELOCK_IOTA_TYPE,
+    });
+
+    const mappedTimelockedObjects = mapTimelockObjects(timelockedObjects || []);
+
+    const totalTimelockedTokens = mappedTimelockedObjects.reduce(
+        (acc, obj) => acc + BigInt(obj.locked.value),
+        BigInt(0),
+    );
+
+    const [formattedTimelockedTokens] = useFormatCoin({
+        balance: totalTimelockedTokens,
+    });
+
+    const totalBalanceBreakdown =
+        BigInt(balance?.totalBalance || 0) +
+        BigInt(totalDelegatedStake || 0) +
+        BigInt(totalTimelockedStaked || 0) +
+        BigInt(totalTimelockedTokens || 0);
+
+    const [formattedTotalBalance] = useFormatCoin({
+        balance: totalBalanceBreakdown,
+    });
+
+    const isLoadingTotalBalance =
+        isLoadingBalance ||
+        isLoadingDelegatedStakes ||
+        isLoadingTimelockedStakeObjects ||
+        isTimelockedObjectsLoading;
+
+    const isTotalBalanceErrored =
+        isBalanceErrored ||
+        isDelegatedStakeErrored ||
+        isTimelockedStakedObjectsErrored ||
+        isTimelockedObjectsError;
+
+    const BALANCE_BREAKDOWN: BalanceBreakdownElement[] = [
+        {
+            keyText: 'Available',
+            value: totalAvailableBalance,
+            supportingLabel: symbol,
+            isLoading: isLoadingBalance,
+            isError: isBalanceErrored,
+            tooltipText: 'IOTA that can be used or transferred immediately.',
```

Comment:
we can simplify the  message to `IOTA that can be used immediately `or `IOTA that can be used right now`